### ISSUE TITLE
Problem with setConstraintUrl and custom validation

### ISSUE THREAD

**Issue by imamovic** [netceteragroup/valdr#85]
If json with validation is loaded from server you can not setup custom validation for same valdr-type.
Custom validation is deleted with validation from server.

Problem is with this: 

```
var addConstraints = function (newConstraints) {
      angular.extend(constraints, newConstraints);
 };

``` 

Need to be user angular.merge();

**Comment by marcelstoer** [comment]
Why do you have tighter validation on the client than on the server (I understand that's what you're doing)? It should be the other way around.

**Comment by imamovic** [comment]
Not exactly, there is and validation on server but it is not been validation, it is custom validation (because of step when data are validated)
